src/py42/_internal/clients/detection_list_user.py

```diff
@@ -0,0 +1,174 @@
+import json
```

Comment:
There are numerous missing `u""` on all these data strings... probably want to make sure we add those.

Reply:
i'll look over those